invadedhouse/run.py

```diff
@@ -19,10 +19,11 @@
 
 
 def main():
-    cprint('   ____                       __         __  __ __                   ', 'cyan')
-    cprint('  /  _/___   _  __ ___  _ ___/ /___  ___/ / / // /___   __ __ ___  ___', 'cyan')
-    cprint(' _/ / / _ \ | |/ // _  `// _  // -_)/ _  / / _  // _ \ / // /(_-</  -_)', 'cyan')
+    cprint('   ____                       __         __  __ __                   ', 'red')
+    cprint('  /  _/___   _  __ ___  _ ___/ /___  ___/ / / // /___   __ __ ___  ___', 'yellow')
+    cprint(' _/ / / _ \ | |/ // _  `// _  // -_)/ _  / / _  // _ \ / // /(_-</  -_)', 'green')
     cprint('/___//_//_/ |___/ \_,_/ \__,_/ __/ \_,_/  /_//_/ ___/ \__,_//___/ \__/ ', 'cyan')
+    print('')
```

Comment:
Cool rainbow design for logo, and added an empty line like you said @jsoconnell

Reply:
nice nice nice, you got the hang of cprint right away! 🌈